bool gesture_found
int32 predicted_class
